ouroboros-consensus-cardano/tools/db-analyser/Block/Cardano.hs

```diff
@@ -51,6 +53,12 @@ import           Block.Byron (Args (..), openGenesisByron)
 import           Block.Shelley (Args (..))
 import           Data.Maybe (fromJust)
 import           HasAnalysis
+import           Ouroboros.Consensus.Protocol.Praos.Translate ()
+import           Ouroboros.Consensus.Protocol.TPraos (TPraos)
+import           Ouroboros.Consensus.Shelley.HFEras ()
+import           Ouroboros.Consensus.Shelley.Ledger.SupportsProtocol ()
+import           Ouroboros.Consensus.Shelley.Node.Praos
+import Ouroboros.Consensus.HeaderValidation (HasAnnTip)
```

Comment:
Style: missing spaces on new `import` statement (and it's relatedly out-of-alphabetical-order)